python/lsst/ip/isr/isrTask.py

```diff
@@ -1211,31 +1214,31 @@ def run(self, ccdExposure, camera=None, bias=None, linearizer=None, crosstalkSou
                             qaMedian = qaStats.getValue(afwMath.MEDIAN)
                             qaStdev = qaStats.getValue(afwMath.STDEVCLIP)
 
-                        self.metadata.set("ISR OSCAN {} MEDIAN".format(amp.getName()), qaMedian)
-                        self.metadata.set("ISR OSCAN {} STDEV".format(amp.getName()), qaStdev)
+                        self.metadata.set(f"ISR OSCAN {amp.getName()} MEDIAN", qaMedian)
+                        self.metadata.set(f"ISR OSCAN {amp.getName()} STDEV", qaStdev)
```

Comment:
I presume there's a good reason for the all caps but it is a bit loud.

Reply:
That's what I found copying other `metadata.set` calls.  I suspect there's an assumption that these may end up as fits header keywords (or `HIERARCH`s), and so upper case?